libraries/eosiolib/contracts/eosio/map.hpp

```diff
@@ -216,29 +246,64 @@ namespace eosio::kv {
                itr_destroy(handle);
          }
 
+         /**
+          * @ingroup keyvaluemap
+          *
+          * @brief Utility function to see if the iterator is valid or is pointing 
+          * at `end` or one past the last or first element.
+          */
          inline bool is_valid() const { return query_status<status::ok>(current_status); }
 
+         /**
+          * @ingroup keyvaluemap
+          *
+          * @brief Function to advance the iterator to the beginning of the map's key value pairs.
+          */
          iterator& seek_to_begin() {
             current_status = static_cast<status>(itr_lower_bound(handle));
             return *this;
          }
 
+         /**
+          * @ingroup keyvaluemap
+          *
+          * @brief Function to advance the iterator to the last element of the map's key value pairs.
+          */
          iterator& seek_to_last() {
             current_status = static_cast<status>(itr_move_to_end(handle));
             current_status = static_cast<status>(itr_prev(handle));
             return *this;
          }
 
+         /**
+          * @ingroup keyvaluemap
+          *
+          * @brief Function to advance the iterator to the element past the last element of the map's key value pairs.
+          */
          iterator& seek_to_end() {
             current_status = static_cast<status>(itr_move_to_end(handle));
             return *this;
          }
 
+         /**
+          * @ingroup keyvaluemap
+          *
+          * @brief This will take a `key_type` key and find the value that is equal to or greater than that key.
+          * @details If no element is greater than or equal to the key the iterator will now hold `end`.
+          * @param key This is the key which you wish to query with.
```

Comment:
@param key This is the key which you wish to query with.
or
@param key The search key.